### PR TITLE
docs: add otelcol FIPS images documentation

### PR DESCRIPTION
**Pull request by perk-sumo**
##### Description

Fill in your description here.

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: docs/security-best-practices.md [hunk 1]
```diff
@@ -358,3 +358,71 @@ fluentd:
...
         readOnly: true
 ```
+## FIPS compliant binaries
+
+We provide FIPS compliant `OpenTelemetry Collector` binaries for your data collection and enrichment.
+
+You can find them in our [ECR Public Gallery](https://gallery.ecr.aws/sumologic/sumologic-otel-collector). FIPS compliant image tags end with `-fips`.
+
+### Helm Chart v3
+Starting with Helm CHart v3 `OpenTelemetry Collector` is a default method of collecting data (except for Prometheus) but still you need set the FIPS comliant images.
```
Comment (sumo-drosiek) [inline docs/security-best-practices.md:368]:
```suggestion
### Helm Chart v3

Starting with Helm CHart v3 `OpenTelemetry Collector` is a default method of collecting data (except for Prometheus) but still you need set the FIPS comliant images.
```

File: docs/security-best-practices.md [hunk 2]
```diff
@@ -358,3 +358,71 @@ fluentd:
         mountPath: /etc/ssl/certs/
         readOnly: true
 ```
+## FIPS compliant binaries
+
+We provide FIPS compliant `OpenTelemetry Collector` binaries for your data collection and enrichment.
```
Comment (sumo-drosiek) [inline docs/security-best-practices.md:363]: Why OpenTelemetry Collector is in ticks?

File: docs/security-best-practices.md (hunk 1 above)
Comment (perk-sumo) [reply docs/security-best-practices.md]: Done, thanks!

File: docs/security-best-practices.md [hunk 3]
```diff
@@ -358,3 +358,71 @@ fluentd:
...
+    image:
+      tag: 0.66.0-sumo-0-fips
+otelgateway:
+  deployment:
+    image:
+      tag: 0.66.0-sumo-0-fips
+```
+
+### FIPS compliant images for Fluent Bit and Fluentd
+Unfortunately this Helm Chart does not provide FIPS compliant binaries for Fluent Bit nor Fluentd.
```
[the remark this answers is not in the document]
Comment (perk-sumo) [reply docs/security-best-practices.md]: Done, thanks!

File: docs/security-best-practices.md [hunk 4]
```diff
@@ -358,3 +358,71 @@ fluentd:
...
+tracesGateway:
+  deployment:
+    image:
+      tag: 0.66.0-sumo-0-fips
+```
+
+### Helm Chart v2
+For Helm Chart v2 please [see here how to switch to `OpenTelemetry Collector`](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v2/deploy/docs/opentelemetry_collector.md).
+
+When you use `OpenTelemetry Collector` you need set the fips comliant images.
```
[the remark this answers is not in the document]
Comment (perk-sumo) [reply docs/security-best-practices.md]: Done, thanks!

File: docs/security-best-practices.md (hunk 2 above)
Comment (perk-sumo) [reply docs/security-best-practices.md]: Good point, I've changed it to bold instead and did the same for Fluent Bit and Fluentd.

File: docs/security-best-practices.md [hunk 5]
```diff
@@ -358,3 +358,74 @@ fluentd:
...
 ```
+## FIPS compliant binaries
+
+We provide FIPS compliant **OpenTelemetry Collector** binaries for your data collection and enrichment.
+
+You can find them in our [ECR Public Gallery](https://gallery.ecr.aws/sumologic/sumologic-otel-collector). FIPS compliant image tags end with `-fips`.
+
+### Helm Chart v3
+
+Starting with Helm Chart v3 **OpenTelemetry Collector** is a default method of collecting data (except for Prometheus) but still you need set the FIPS comliant images.
```
Comment (swiatekm-sumo) [inline docs/security-best-practices.md:369]:
```suggestion
Starting with Helm Chart v3 **OpenTelemetry Collector** is a default method of collecting data (except for Prometheus) but still you need set the FIPS compliant images.
```

**Review by swiatekm-sumo — COMMENTED** [review]
Do we want to claim this before explicitly claiming FIPS compliance for otel distro?

File: docs/security-best-practices.md (hunk 5 above)
Comment (perk-sumo) [reply docs/security-best-practices.md]: 👍

**Comment by perk-sumo** [comment]
> Do we want to claim this before explicitly claiming FIPS compliance for otel distro?

I don't think we need to wait for that, they are compliant, we "just" need to add our claim doc for it.

**Pull request merged**